python_modules/libraries/dagster-gcp/dagster_gcp/bigquery/resources.py

```diff
@@ -96,3 +101,39 @@ def bigquery_resource(context):
     bq_resource = BigQueryResource.from_resource_context(context)
     with bq_resource.get_client() as client:
         yield client
+
+
+def get_data_freshness(
```

Comment:
I think we can find a name that's more explicitly communicates what data this returns, e.g. `fetch_last_updated_timestamps` or `fetch_modified_timestamps`.